gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -19,74 +19,136 @@
     "use strict";
 
     var fluid = require("infusion"),
-        gpii = fluid.registerNamespace("gpii");
+        gpii = fluid.registerNamespace("gpii"),
+        $ = fluid.registerNamespace("jQuery");
 
     fluid.registerNamespace("gpii.contextManager");
 
     fluid.defaults("gpii.contextManager", {
         gradeNames: ["fluid.eventedComponent", "autoInit"],
+        members: {
+            currentContext: {},
```

Comment:
currentContext should be attached to the component's model instead, and this should be expressed as a "fluid.modelRelayComponent". Changes to it should be triggered using its ChangeApplier, and listened to via modelListeners.